android/src/main/java/com/wenkesj/voice/VoiceModule.java

```diff
@@ -30,49 +31,62 @@
   final ReactApplicationContext reactContext;
   private SpeechRecognizer speech = null;
   private boolean isRecognizing = false;
+  private String locale = null;
+  private int numberOfBreakingSentence = 0;
```

Comment:
Can you elaborate the meaning of this variable naming?

Reply:
Android can break recognizing if a user stops speaking. Our purpose is recognizing before we implicitly stop recognizing. Thus, this variable means that how many times a user stops speaking between start and end of recognizing. Um... `numberOfStopSpeakingInSentence` is more better?

Reply:
Could you put that explanation in our code as a comment?